util/hash_attributes.ts

```diff
@@ -0,0 +1,61 @@
+import { getBinaryTreeData } from './binaryTreeBranch';
+
+export const HASH_ATTRIBUTES: Attribute[] = [
+    {
+        descriptionShort: "Leading 0's",
+        description: "Length of the streak of 0's at the start of the binary number representation.",
```

Comment:
I left `description` in for now, we may use it?

Reply:
yeah definetly